src/smt/set_defaults.cpp

```diff
@@ -62,6 +62,12 @@ void SetDefaults::setDefaults(LogicInfo& logic, Options& opts)
 
 void SetDefaults::setDefaultsPre(Options& opts)
 {
+  // internal-only options
+  if (opts.smt.unsatCoresMode == options::UnsatCoresMode::PP_ONLY)
+  {
+    throw OptionException(
+        std::string("Unsat core mode pp-only is for internal use only."));
```

Comment:
What would probably be better is to use the string representation of `options::UnsatCoresMode::PP_ONLY` instead of the `pp-only` string.

Reply:
Aren't they the same? `pp-only` is the what string is used in `set-option`.